Milliner/tests/Islandora/Milliner/Tests/DrupalEntityConverterTest.php

```diff
@@ -43,10 +43,6 @@ public function setUp()
         $this->logger_prophecy = $this->prophet->prophesize('Psr\Log\LoggerInterface');
     }
 
-    /**
-     * @covers ::__construct
-     * @covers ::convert
-     */
```

Comment:
So this is actually for PHPUnit and it define which functions this test covers. You can do `@coverts \Islandora\Milliner\Converter\DrupalEntityConverter::convert`, or if you have defined the class name [here](https://github.com/bseeger/Crayfish/blob/d8dc8fe7aec42d2fd76289d7fdf05643a9e9c0fa/Milliner/tests/Islandora/Milliner/Tests/DrupalEntityConverterTest.php#L13) you can just do `::functionName`.

So please put this back, and you can add some for your new test.

Reply:
Here is the documentation on the options.
https://phpunit.de/manual/current/en/appendixes.annotations.html

Reply:
I guess removing the `@covers` had it check the entire class, so you can disregard this comment.

Reply:
Yeah, that's what I found. When I took those statements out it seemed that it actually started looking at the whole file and checked everything.  

The other thing I found was that this original test function _did not_ actually cover the `convert` method at all. It covered the `convertJsonld` method.  

This is a phpunit crash course for me, so thanks for the link.

Reply:
@whikloj  - If you're okay with this as is, I think you might need to remove the request for a change before this can be merged?